generate/hcl.go

```
@@ -157,12 +157,16 @@ func FromHCL(fs afero.Fs, path string, opt Options) (*graph.Graph, error) {
 		return nil, err
 	}
 
-	err = mutate(g, opt)
-	if err != nil {
-		return nil, err
+	if opt.Connections {
+		err = mutate(g, opt)
+		if err != nil {
+			return nil, err
+		}
 	}
 
-	cleanHangingEdges(g, opt)
+	if opt.Clean {
+		cleanHangingEdges(g, opt)
```

`cleanHangingEdges` can silently fail. It would be nice to catch the error ?